tests/fsharp/typecheck/sigs/neg110.bsl

```diff
@@ -1,2 +1 @@
 neg110.fs(5,3,5,15): typecheck error FS3242: This type does not inherit Attribute, it will not work correctly with other .NET languages.
```

Comment:
```suggestion
neg110.fs(5,3,5,15): typecheck error FS3242: This type does not inherit Attribute, it will not work correctly with other .NET languages.

```